docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -56,9 +56,26 @@ To search all data streams and indices in a cluster, use
 [[eql-search-api-query-params]]
 ==== {api-query-parms-title}
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+`allow_no_indices`::
+(Optional, boolean)
++
+NOTE: This parameter's behavior differs from the `allow_no_indices` parameter
+used in other <<multi-index,multi-target APIs>>.
++
+If `false`, any wildcard expression, <<indices-aliases,index alias>>, or `_all`
+value that targets only missing or closed indices returns an error. This applies
+even if the request targets other open indices. For example, if an index starts
+with `foo` but no index starts with `bar`, a request targeting `foo*,bar*`
+returns an error.
```

Comment:
```suggestion
If `false`, requests return an error if any wildcard expression, <<indices-aliases,index alias>>, or `_all`
value targets only missing or closed indices. This behavior applies
even if the request targets other open indices. For example, a request
targeting `foo*,bar*` returns an error if any index starts with `foo`, but no
index starts with `bar`.
```

Reply:
I moved "requests return an error" to the beginning of the first sentence, which I think reads easier. "You'll get an error if X, Y, or Z" feels more natural than "If X, Y, or Z, you'll get an error." Apologies for such subjective logic, but I kept stumbling on that first sentence.